useScroll/ScrollbarPlaceHolder.js

```diff
@@ -0,0 +1,48 @@
+import Spotlight from '@enact/spotlight';
+import Spottable from '@enact/spotlight/Spottable';
+import {useCallback, useEffect, useState} from 'react';
+
+const SpotlightPlaceholder = Spottable('div');
+
+/**
+ * ScrollbarPlaceHolder component.
+ *
+ * @class ScrollbarPlaceHolder
+ * @memberof sandstone/useScroll
+ * @ui
+ * @private
+ */
+const ScrollbarPlaceHolder = () => {
```

Comment:
Let's rename this to `ScrollbarPlaceholder`.

Reply:
I fixed